### DISCUSSION TITLE
lesson 6: Deploying to a local blockchain III (scripting)

### DISCUSSION THREAD

**Discussion by callyongg** [Cyfrin/foundry-full-course-cu#2005] [Q&A]
Hello, I need help for my lib/forge-std folder
![image](https://github.com/user-attachments/assets/abb05540-6e79-407b-bccf-a6916c3a41b9)
I don't have that folder in my vscode. Please give me a solution

**Comment by Podziuu** [comment]
Hello try this command in your working directory to install forge-std library `forge install foundry-rs/forge-std --no-commit`

**Reply by callyongg** [reply]
oh yeah, thank you for helping me

**Comment by EngrPips** [comment]
Hello @yongcen, try what is suggested by @Podziuu. it should solve your problem.

**Reply by callyongg** [reply]
okay, it works. thank you

**Reply by EngrPips** [reply]
Awesome, I'm glad you figured it out. Keep crushing it.
